src/parser/mod.rs

```diff
@@ -1298,7 +1301,14 @@ impl<'a> Parser<'a> {
             }));
         }
 
-        let args = self.parse_function_argument_list()?;
+        let mut args = self.parse_function_argument_list()?;
+        let mut parameters = FunctionArguments::None;
+        // ClickHouse aggregation support parametric functions like `quantile(0.5)(x)`
+        // which (0.5) is a parameter to the function.
+        if dialect_of!(self is ClickHouseDialect) && self.consume_token(&Token::LParen) {
```

Comment:
```suggestion
        if dialect_of!(self is ClickHouseDialect | GenericDialect) && self.consume_token(&Token::LParen) {
```
we can include generic dialect if no conflicts